### ISSUE TITLE
Translations broken in latest build

### ISSUE THREAD

**Issue by HaakonH** [ngx-rocket/generator-ngx-rocket#181]
## I'm submitting a...

[X] Bug report  
[ ] Feature request
[ ] Documentation issue or request

## Current behavior

If you create a new ngx-rocket project and generate it and then do npm start and change the language to french or change a string in the translations you will note see translations.

## Expected behavior

That translations would work as in previous builds or that if they don't work there would be an error in the console.

## Minimal reproduction of the problem with instructions

1. Update ngx-rocket generator
2. Create a new project folder, open that folder in your cmd window. Run - yongx-rocket
3. Accept defaults.
4. After installs and the npm update run - npm start
5. Open your browser to the home page, change the language to french and notice that the language doesn't change, in fact it is just displaying the tag names and not the actual translations.

## Environment

```
- generator version: 1.5.2
- node version: 6.9.1
- npm version: 3.10.8
- OS:  Windows 10
```

Others:
Here is a screen shot - http://haakon.io/assets/images/translations.PNG

**Comment by sinedied** [comment]
Seems related to #165, but I cannot reproduce it with a fresh install.

Could you try this and tell me if it works?
```sh
npm rm -g @ngx-rocket/cli
npm cache clean
npm i -g @ngx-rocket/cli
```
This should make sure the CLI fetch the latest generator version when installing.

I'm not sure why the generator dependency would not have been updated properly though 😕

**Comment by HaakonH** [comment]
I tried running the scripts as directed. I then did a npm install and npm update. Then I did npm start and when the app launched I see the same issue.

**Comment by sinedied** [comment]
Can you post here the result of `npm ls -g --depth=1` on your system?

Also I have read again the description of your issue, did you use the generator directly with `yo ngx-rocket` or the CLI with `ngx new` (as you reported the CLI version)?

If it's the generator, you can try:
```
npm rm -g generator-ngx-rocket
npm cache clean
npm i -g generator-ngx-rocket@latest
```

**Comment by HaakonH** [comment]
I ran the generator using the "yo ngx-rocket" command and used all the default properties except I didn't choose authentication.

Here is the result of the depth script:

C:\Users\haakon\AppData\Roaming\npm
+-- @angular/cli@1.4.3
| +-- @angular-devkit/build-optimizer@0.0.21
| +-- @angular-devkit/schematics@0.0.24
| +-- @ngtools/json-schema@1.1.0
| +-- @ngtools/webpack@1.7.1
| +-- @schematics/angular@0.0.36
| +-- autoprefixer@6.7.7
| +-- chalk@2.1.0
| +-- circular-dependency-plugin@3.0.0
| +-- common-tags@1.4.0
| +-- copy-webpack-plugin@4.0.1
| +-- core-object@3.1.5
| +-- css-loader@0.28.7
| +-- cssnano@3.10.0
| +-- denodeify@1.2.1
| +-- ember-cli-string-utils@1.1.0
| +-- exports-loader@0.6.4
| +-- extract-text-webpack-plugin@3.0.0
| +-- file-loader@0.10.1
| +-- fs-extra@4.0.2
| +-- get-caller-file@1.0.2
| +-- glob@7.1.2
| +-- heimdalljs@0.2.5
| +-- heimdalljs-logger@0.1.9
| +-- html-webpack-plugin@2.30.1
| +-- istanbul-instrumenter-loader@2.0.0
| +-- karma-source-map-support@1.2.0
| +-- less@2.7.2
| +-- less-loader@4.0.5
| +-- license-webpack-plugin@1.0.1
| +-- lodash@4.17.4
| +-- memory-fs@0.4.1
| +-- node-modules-path@1.0.1
| +-- node-sass@4.5.3
| +-- nopt@4.0.1
| +-- opn@5.1.0
| +-- portfinder@1.0.13
| +-- postcss-loader@1.3.3
| +-- postcss-url@5.1.2
| +-- raw-loader@0.5.1
| +-- resolve@1.4.0
| +-- rxjs@5.4.3
| +-- sass-loader@6.0.6
| +-- semver@5.4.1
| +-- silent-error@1.1.0
| +-- source-map-loader@0.2.1
| +-- source-map-support@0.4.18
| +-- style-loader@0.13.2
| +-- stylus@0.54.5
| +-- stylus-loader@3.0.1
| +-- typescript@2.5.2
| +-- url-loader@0.5.9
| +-- webpack@3.5.6
| +-- webpack-concat-plugin@1.4.0
| +-- webpack-dev-middleware@1.12.0
| +-- webpack-dev-server@2.7.1
| +-- webpack-merge@4.1.0
| `-- zone.js@0.8.17
+-- @ngx-rocket/cli@1.5.2
| +-- @ngx-rocket/ascii-logo@1.1.0
| +-- chalk@2.3.0
| +-- conf@1.3.1
| +-- figures@2.0.0
| +-- generator-ngx-rocket@2.4.0
| +-- generator-ngx-rocket-addon@1.3.0
| +-- inquirer@4.0.0
| +-- minimist@1.2.0
| +-- update-notifier@2.3.0
| `-- yeoman-environment@2.0.5
+-- bower@1.8.2
+-- express-generator@4.15.5
| +-- commander@2.11.0
| +-- ejs@2.5.7
| +-- mkdirp@0.5.1
| `-- sorted-object@2.0.1
+-- generator-angular@0.16.0
| +-- chalk@1.1.3
| +-- wiredep@2.2.2
| +-- yeoman-generator@0.16.0
| `-- yosay@1.2.1
+-- generator-fountain-angular2@1.0.0-rc2
| +-- fountain-generator@1.0.0-rc2
| `-- generator-fountain-gulp@1.0.0-rc2
+-- generator-fountain-webapp@1.0.0-rc2
| +-- chalk@1.1.3
| +-- fountain-generator@1.0.0-rc2
| +-- generator-fountain-angular1@1.0.0-rc2
| +-- generator-fountain-angular2@1.0.0-rc2
| +-- generator-fountain-react@1.0.0-rc2
| +-- generator-fountain-vue@1.0.0-rc2
| `-- yosay@1.2.1
+-- generator-karma@2.0.0
| `-- yeoman-generator@0.22.6
+-- generator-ngx-rocket@2.1.3
| +-- @ngx-rocket/ascii-logo@1.1.0
| +-- @ngx-rocket/core@2.1.0
| +-- chalk@2.1.0
| `-- insight@0.8.4
+-- gulp@3.9.1
| +-- archy@1.0.0
| +-- chalk@1.1.3
| +-- deprecated@0.0.1
| +-- gulp-util@3.0.7
| +-- interpret@1.0.1
| +-- liftoff@2.3.0
| +-- minimist@1.2.0
| +-- orchestrator@0.3.8
| +-- pretty-hrtime@1.0.3
| +-- semver@4.3.6
| +-- tildify@1.2.0
| +-- v8flags@2.0.11
| `-- vinyl-fs@0.3.14
+-- gulp-cli@1.2.2
| +-- archy@1.0.0
| +-- chalk@1.1.3
| +-- fancy-log@1.3.0
| +-- gulplog@1.0.0
| +-- interpret@1.0.1
| +-- liftoff@2.3.0
| +-- lodash.isfunction@3.0.8
| +-- lodash.isplainobject@4.0.6
| +-- lodash.isstring@4.0.1
| +-- lodash.sortby@4.7.0
| +-- matchdep@1.0.1
| +-- mute-stdout@1.0.0
| +-- pretty-hrtime@1.0.3
| +-- semver-greatest-satisfied-range@1.0.0
| +-- tildify@1.2.0
| +-- v8flags@2.0.11
| +-- wreck@6.3.0
| `-- yargs@3.32.0
+-- rimraf@2.6.2
| `-- glob@7.1.2
`-- yo@2.0.0
  +-- async@2.5.0
  +-- chalk@1.1.3
  +-- cli-list@0.2.0
  +-- configstore@3.1.1
  +-- cross-spawn@5.1.0
  +-- figures@2.0.0
  +-- fullname@3.3.0
  +-- got@6.7.1
  +-- humanize-string@1.0.1
  +-- inquirer@3.3.0
  +-- insight@0.8.4
  +-- lodash@4.17.4
  +-- meow@3.7.0
  +-- npm-keyword@4.2.0
  +-- opn@4.0.2
  +-- package-json@2.4.0
  +-- parse-help@0.1.1
  +-- read-pkg-up@2.0.0
  +-- root-check@1.0.0
  +-- sort-on@2.0.0
  +-- string-length@1.0.1
  +-- tabtab@1.3.2
  +-- titleize@1.0.0
  +-- update-notifier@2.2.0
  +-- user-home@2.0.0
  +-- yeoman-character@1.1.0
  +-- yeoman-doctor@2.1.0
  +-- yeoman-environment@2.0.3
  `-- yosay@2.0.1

**Comment by HaakonH** [comment]
Update. I have create two new test projects. One using 'yo ngx-rocket' and one using 'ngx new'.

The 'ngx new' project works just fine.

The 'yo ngx-rocket' builds and shows no errors but the translations are broken.

I will use the first until I hear that the yeoman generator build is fixed.

**Comment by sinedied** [comment]
Thanks for the feedback. It seems an old version of the generator is installed on your system from your log: `generator-ngx-rocket@2.1.3, and the translations bug was fixed in `@2.4.0`.

The CLI comes with its own generator version as a dependency, that explains why it worked properly.

Normally these command should make sure that the latest generator is installed:
```
npm remove -g generator-ngx-rocket
npm cache clean
npm install -g generator-ngx-rocket@latest
```
You can check the installed version then with `npm ls -g generator-ngx-rocket --depth=0` to see if it's version `2.4.0`.
